### ISSUE TITLE
mat2doc questions

### ISSUE THREAD

**Issue by susnak** [ltfat/mat2doc#1]
I have been tinkering with both mat2doc.py and mat2docnotes.py for some time and I have few questions..

  1) Is there a reason why mat2doc is not a Python module? There are parts of code which could be shared    between both scripts...

  2) Is it really necessary for the findMat2docDir function to do such directory crawling? The only scenario I can imagine it can be usefull is when a single file or a subdirectory should be processed.

**Comment by psoenderg** [comment]
No. 1: No, no reason. The code is just not very pretty, and this is one of the cases.

No. 2: I think your theory is right, at some point I wanted to make mat2doc so that you could process just a single file, and this is probably the reason why the code is still there. Now mat2doc will just process the updated files, so there is no need to be able to handle a single file.

**Comment by susnak** [comment]
Ok. Another thing. Would it make sense to allow multiple targets of the same type? E.g. having mat2doc/html and mat2doc/html2 etc.? The list of available targets could be determined at the beginning of mat2doc[notes]
This could be more versatile than shadowing the configuration files..
